### PR TITLE
Show loading view while loading deleted objects

### LINKED ISSUE #3332: The trash bin briefly shows `You don't have any deleted calendars or events` until loading has finished.
It should instead indicate that deleted items are currently loaded.

### PR DESCRIPTION
**Pull request by raimund-schluessler**
loading

Closes #3332.

### CODE REVIEW

File: src/components/AppNavigation/CalendarList/Trashbin.vue [hunk 1]
```diff
@@ -29,55 +29,62 @@
...
-						<tr v-for="item in items" :key="item.url">
-							<td>
-								<div class="item">
-									<div>
-										<div
-											class="color-dot"
-											:style="{ 'background-color': item.color }" />
-									</div>
+					<EmptyContent v-if="loading" icon="icon-loading">
+						{{ t('calendar', 'Loading deleted calendars, events and tasks.') }}
```
Comment (tcitworld) [inline src/components/AppNavigation/CalendarList/Trashbin.vue:33]:
```suggestion
						{{ t('calendar', 'Loading deleted elements.') }}
```

File: src/components/AppNavigation/CalendarList/Trashbin.vue [hunk 2]
```diff
@@ -29,55 +29,62 @@
...
+					</EmptyContent>
+					<EmptyContent v-else-if="!items.length" icon="icon-delete">
+						{{ t('calendar', 'You do not have any deleted calendars, events or tasks.') }}
```
Comment (tcitworld) [inline src/components/AppNavigation/CalendarList/Trashbin.vue:36]:
```suggestion
						{{ t('calendar', 'You do not have any deleted elements.') }}
```

**Review by tcitworld — APPROVED** [review]
Just a wording suggestion.

File: src/components/AppNavigation/CalendarList/Trashbin.vue (hunk 1 above)
Comment (raimund-schluessler) [reply src/components/AppNavigation/CalendarList/Trashbin.vue:33]: Fine with me.

**Review by ChristophWurst — APPROVED** [review]

**Comment by ChristophWurst** [comment]
the only thing that I don't like personally is how the font became bold. it looks like the same weight and size as the header is used, right? could this be lowered?

**Comment by raimund-schluessler** [comment]
> 
> 
> the only thing that I don't like personally is how the font became bold. it looks like the same weight and size as the header is used, right? could this be lowered?

This comes from the EmptyContent component from nextcloud/vue in combination with a server rule for `h2` which makes headlines bold. But I don't mind. Feel free to adjust it.

**Comment by raimund-schluessler** [comment]
> the only thing that I don't like personally is how the font became bold. it looks like the same weight and size as the header is used, right? could this be lowered?

How about we don't show the trash bin header at all while loading the items, but only the empty content header with the icon?

**Comment by st3iny** [comment]
> > the only thing that I don't like personally is how the font became bold. it looks like the same weight and size as the header is used, right? could this be lowered?
> 
> This comes from the EmptyContent component from nextcloud/vue in combination with a server rule for `h2` which makes headlines bold. But I don't mind. Feel free to adjust it.

I fixed that and also lowered the top margin a bit to make the loading and empty messages look more centered.

Screenshot 2021-08-06 at 11-16-30 August 2021 - Calendar - Nextcloud
Screenshot 2021-08-06 at 11-16-54 August 2021 - Calendar - Nextcloud

**Review by st3iny — APPROVED** [review]

**Pull request merged**